unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/PoolManagerFactory.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
+
+namespace DCL
+{
+    public static class PoolManagerFactory
```

Comment:
I think we should avoid using static classes, in this case, a factory. It makes it harder to extend, scale or mock

Reply:
Yeah, I agree about removing the static classes, although they are fine for containers of pure methods like this that don't need mocking (like our `Utils` set of classes). The idea of this class was to move the pool prewarming out of `SceneController` as it shouldn't be its responsibility. I'm going to think how to improve this, but will do on another PR.